packages/web/src/components/comments/CommentBlock.tsx

```diff
@@ -167,6 +167,7 @@ const CommentBlockInternal = (
               isPreview={isPreview}
               mentions={mentions}
               commentId={commentId}
+              duration={track.duration}
```

Comment:
oh oops. I went back and forth on this when I was updating CommentText to be usable outside of the section context and I forgot to add this back in. Thanks for catching it.